### ISSUE TITLE
Unused void attributes must be prohibited

### ISSUE THREAD

**Issue by yegor256** [objectionary/lints#24]
Consider this code:

```
[x] > foo
  42 > bar
```

Here, the `x` void attribute has no meaning and must be removed, because it's not used anywhere in the object. Of course, if there are atoms inside the object, we must not complain, like in this program, which is valid:

```
[x] > foo
  [] > bar /int
```

**Comment by maxonfjvipon** [comment]
@yegor256 what about if this void attribute is used by other object outside of scope of current object
```
[] > main
  [x] > foo
  [f] > bar
    (^.foo 5).x > @
```

**Comment by yegor256** [comment]
@maxonfjvipon I suggest we still complain about it. If it's not used _by the object_, it's a code smell
